howtos/technical-writing-cheatsheet.md

```diff
@@ -10,42 +10,41 @@ Our primary goal in documentation is to achieve organization, clarity, and direc
 
 ## Grammar
 
-| Subject | Practice | Avoid | Example/Use |
-| -- | -- | -- | -- |
-| Bolding | Bold when referring to button names or items to select. | Click "Create New Diagram." | Click **Create New Diagram** under the **Diagrams** tab. |
-| Italics | Use when applying emphasis to a word. | Click *Create New Diagram*. | Click **Create New Diagram**. |
-| Numbers | Write whole numbers one through nine in full. Write whole numbers 10 and upwards as numerals. | In this example, we will create 1 diagram. | In this example, we will create one diagram. |
-| Spelling | Default to American spelling and US English. Visit the [Oxford American Dictionary](https://www.oxfordreference.com/view/10.1093/acref/9780195392883.001.0001/acref-9780195392883) for details. | Analyse<br><br>Bernd Rücker | Analyze<br><br>Bernd Ruecker |
-| Voice/Tense | Second person, [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/#:~:text=Active%20voice%20means%20that%20a,it%20isn't%20that%20simple.). | I, me, my.<br><br>The computer is turned on by pressing the power button. | You, your.<br><br>Press the power button to turn on the computer. |
+| Subject     | Practice                                                                                                                                                                                        | Avoid                                                                     | Example/Use                                                       |
```

Comment:
I could see the formatting of these tables being controversial. They look much nicer in an editor, with word-wrap turned off: 

<img width="1557" alt="image" src="https://user-images.githubusercontent.com/1627089/170371940-34a0154e-f1e3-4e89-badc-9c5915d8cf0b.png">

If we collectively don't like this format, we can revert to the compact form.

There is no setting for "compact tables" -- [prettier doesn't want to have an opinion on that](https://github.com/prettier/prettier/issues/5651) -- but I _can_ make prettier use the compact version for these tables by setting [`proseWrap` to `never`](https://prettier.io/docs/en/options.html#prose-wrap). That does have the side effect of never wrapping long paragraphs in Markdown, but it looks like we don't wrap paragraphs intentionally anyway.

Reply:
From a technical writing perspective, this looks great to me.

Reply:
@christinaausley I'm done waiting for reviews on this one -- mind giving me another green review so I can merge, and if anyone comes along expressing dislike for the table formatting, we can revisit?